### PR TITLE
Task-58643: Related the attached file thumbnails to attachment file extension

### PR DESCRIPTION
**Pull request by sofyenne**
ISSUE : when user create process and attached a file all attached file thumbnails displayed as media , the problem is when clean name of uploaded document   can't extract the extension from the name .
FIX : modified clean name method to correctly clean the name file

### CODE REVIEW

File: core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java [hunk 1]
```diff
@@ -701,16 +701,23 @@ public static String cleanString(String str) {
...
     for (int i = 0; i < oldName.length(); i++) {
       char currentChar = oldName.charAt(i);
       if (specialChar.indexOf(currentChar) > -1) {
-        ret.append('-');
+        ret.append('_');
       } else {
         ret.append(currentChar);
       }
     }
+    if(ret.indexOf(".") > -1 && ret.indexOf(".") != ret.lastIndexOf(".") ){
```
Comment (ahamdi) [inline core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java]: You can just substring the extension in the beginning of the cleanup process and then do the cleanup includin hg the dot (.) character.

File: core/services/src/test/java/org/exoplatform/services/cms/impl/UtilsTest.java [hunk 2]
```diff
@@ -48,27 +49,29 @@ public void testCleanName() {
...
-    assertEquals(titleClean12,"test-test");
-    assertEquals(titleClean13,"test-test");
-    assertEquals(titleClean14,"test-test");
-    assertEquals(titleClean15,"test-test");
-    assertEquals(titleClean16,"test-test");
-    assertEquals(titleClean17,"test-test");
-    assertEquals(titleClean18,"test-test");
-    assertEquals(titleClean19,"test-test");
-    assertEquals(titleClean20,"test-test");
+    assertEquals(titleClean1,"test _ test");
```
Comment (ahamdi) [inline core/services/src/test/java/org/exoplatform/services/cms/impl/UtilsTest.java]: why there are spaces before/after filename ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java (hunk 1 above)
Comment (sofyenne) [reply core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java]: Done !

File: core/services/src/test/java/org/exoplatform/services/cms/impl/UtilsTest.java (hunk 2 above)
Comment (sofyenne) [reply core/services/src/test/java/org/exoplatform/services/cms/impl/UtilsTest.java]: done ! space removed !

**Review by ahamdi — APPROVED** [review]

**Pull request merged**